lib/codequest_pipes/pipe.rb

```diff
@@ -26,12 +26,18 @@ def self.call(ctx)
       _validate_ctx(_provided_context_elements, ctx)
     end
 
-    def self.require_context(*args)
-      _required_context_elements.push(*args)
+    def self.require_context(*args, **kwargs)
+      _required_context_elements.merge!(
+        **args.map { |a| [a, nil] }.to_h,
+        **kwargs
+      )
     end
 
-    def self.provide_context(*args)
-      _provided_context_elements.push(*args)
+    def self.provide_context(*args, **kwargs)
+      _provided_context_elements.merge!(
```

Comment:
Operations performed on `provide` and `require` arguments are basically the same. How about extracting those operations to one method?

Reply:
👍